imagehash/__init__.py

```diff
@@ -45,7 +45,7 @@ def binary_array_to_hex(arr):
 	return "".join(s)
 
 def binary_array_to_int(arr):
-	return sum([2**(i % 8) for i,v in enumerate(arr.flatten()) if v])
+	return sum([2**i for i,v in enumerate(arr.flatten()) if v])
```

Comment:
Please remove this change from this pull request because it is about another issue (https://github.com/JohannesBuchner/imagehash/issues/5). I know this function is problematic and will address it. Lets keep that separate though.